src/lib/export/exporters/exporters.test.ts

```diff
@@ -986,4 +987,39 @@ terminal_colors:
 
     expect(config).toBe(expected);
   });
+
+  it("to Tabby", () => {
+    const config = toTabby(someRecipe);
+    // prettier-ignore
+    const expected = `# Copy the configuration below and add it under
+# terminal.customColorSchemes in the appropriate config file:
+# Linux: ~/.config/tabby/config.yaml
+# macOS: ~/Library/Application Support/tabby/config.yaml
+# Windows: ~\\AppData\\Roaming\\tabby\\config.yaml
```

Comment:
The double backslashes here should be single backslashes, right?

Reply:
The unit tests fail of the backslashes aren't doubled, presumably because that's the escape character.